tests/integration/base.py

```diff
@@ -19,7 +19,7 @@ class CommandResult(NamedTuple):
     stderr: str
 
 
-TIMEOUT = 300
+TIMEOUT = 3600
```

Comment:
This one is a little tricker. Since `run_command` uses this constant as a default value and is defined outside the `IntegBase` class i basically have two options as i see it.

1. Move `run_command` into `IntegBase` and change all invocations of `run_command` to `self.run_command` with KV-arg timeout defaulting to the new `command_timeout` field.

2. Passing timeout  kv-argument with default from the new `command_timeout` field. everywhere where `run_command` is invoked.

Both these changes requires a lot of changes for very little gain. Or am i missing something here? 🤔

Reply:
@moelasmar ☝️

Reply:
Ok. Can you add a TODO comment to refactor this part later.